Makefile.common

```diff
@@ -0,0 +1,252 @@
+# Copyright 2018 The Prometheus Authors
```

Comment:
It doesn't seem to be the same version than the [original Makefile.common](https://github.com/prometheus/prometheus/blob/master/Makefile.common). The rule is that any change to Makefile.common needs to be opened against the github.com/prometheus/prometheus repository. Then we have automation in place that automatically pushes changes to the other repositories when the copies don't match with the source anymore.

Reply:
this change adds `DOCKERFILE_PATH` makefile variable, so if I add the PR to prometheus/prometheus will also have to update the corresponding Makefiles of each repository using Makefile.common.  right?

Also, can we add prombench to the automation where the Makefile.common changes are pushed?

Reply:
> this change adds DOCKERFILE_PATH makefile variable, so if I add the PR to prometheus/prometheus will also have to update the corresponding Makefiles of each repository using Makefile.common. right?

Yes this change needs to go first to prometheus/prometheus.

> Also, can we add prombench to the automation where the Makefile.common changes are pushed?

This will be done automatically.